sync/awsdatabricks.py

```diff
@@ -388,10 +382,67 @@ def write_file(body: bytes):
                     "Reservations": [*removed_instances, *updated_instances, *new_instances]
                 }
 
-            write_file(orjson.dumps(instances))
+            # Now record the ebs volumes
+            volumes = _get_ebs_volumes(cluster_id, ec2)
+
+            for v in volumes:
+                if v["VolumeId"] not in recorded_volume_id_set:
+                    recorded_volume_id_set.add(v["VolumeId"])
+                    recorded_volumes.append(v)
 
+            instances["Volumes"] = recorded_volumes
```

Comment:
This solution won't extend to Azure Databricks because the instances in a cluster report is an array instead of a dict: https://github.com/synccomputingcode/syncsparkpy/blob/c5575c873acc150e9f5d931f0adca0c8296d2122/sync/models.py#L110-L111
We could make it a dict, but seems better to put volumes under "volumes" at the top level if only for better naming.

Reply:
Gotcha, that's something I was considering doing, and I think I like it more. It maintains some parity with Azure (though idk if a separate call like this is needed for Azure), but it also keeps the existing structures unchanged which has some benefit to mental load.

Reply:
You said it better than me!